### DISCUSSION TITLE
If still struggling with a GUI why not a (local) web app?

### DISCUSSION THREAD

**Discussion by joaogelado** [ThioJoe/YT-Spammer-Purge#155] [Ideas]
What if, instead of a GUI, we had a local web application running. Easier than programming an entire GUI (i think. HTML takes a big part of the reason) and there is a uncountable quantity of web frameworks.

**Comment by joaogelado** [comment]
also i'll try to make this but with node.js and next.js soon™️

**Reply by TechStudent10** [reply]
You could use PyWebview.

**Reply by joaogelado** [reply]
i'm more confortable with node.js

**Reply by TechStudent10** [reply]
Okay, but this program is made in Python, so using a Python-based framework/library would make it easier.

**Comment by ThioJoe** [comment]
Eh to be honest I don't see too much added value in a GUI at this point. The script already only requires typing in some numbers and pasting in URLs. And it's way easier to add features with it being command line.
